strider/server.py

```diff
@@ -186,7 +186,7 @@ async def catch_exceptions_middleware(request: Request, call_next):
 
 
 @APP.on_event("startup")
-async def get_kp_registry():
+async def refresh_kp_registry():
```

Comment:
If we change the name of this function, what are we calling now in line 682?

Reply:
We are now correctly calling line 48

Reply:
Ah okay, I missed that import. Thanks.